src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -48,7 +48,7 @@ namespace Microsoft.PowerShell
     internal static class ConsoleControl
     {
 #if !UNIX
-#region structs
+        #region structs
```

Comment:
this format seems to contradict https://github.com/PowerShell/PowerShell/pull/9402/files#diff-d60cf147d07e56d9bad113594e47c7f0R275

Reply:
I see only one strange case in PSTelemetryMethods.cs there. Here all looks ok.